deploy/charts/cert-manager/webhook/templates/ca-sync.yaml

```diff
@@ -128,6 +128,8 @@ metadata:
     chart: {{ include "webhook.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+
+{{- if .Values.global.rbac.create -}}
```

Comment:
You probably want to remove the first `-` (after `{{`) else Helm's templating is eating the whitespace *before* this line, causing the error here: https://github.com/jetstack/cert-manager/pull/1339/files#diff-ec2bdfe7739de2a6a20cf29b04138efaR565

Reply:
Nice catch.  Bad habits on my part.  Thanks.

Reply:
That seems to have fixed the tests.  Thanks again.